util/pace/util/namelist.py

```diff
@@ -105,6 +105,7 @@ class NamelistDefaults:
     tice = 273.16  # set tice = 165. to turn off ice - phase phys (kessler emulator)
     alin = 842.0  # "a" in lin1983
     clin = 4.8  # "c" in lin 1983, 4.8 -- > 6. (to ehance ql -- > qs)
+    stretch_grid = False
```

Comment:
Does this actually need to be added to our Namelist class? It's a little misleading to put it here, because the initialization code doesn't read from the namelist at all. I think you removed the place where the dycore was accessing this by putting it on the grid data containers instead.